roles/fluentd_master/meta/main.yml

```diff
@@ -11,5 +11,5 @@ galaxy_info:
     - 7
   categories:
   - monitoring
-  dependencies:
-  - openshift_facts
+dependencies:
+- { role: openshift_facts }
```

Comment:
are the fluentd_* roles even in use anymore? part of me wants to say that we should just remove them...

@wshearn is Ops using the fluentd_* roles? If so, any objections to moving them over to openshift_tools?